test.properties

```diff
@@ -4666,6 +4740,83 @@
 
     test.release.bundle[portal-release-tomcat]=true
 
+    #
+<<<<<<< HEAD
```

Comment:
@CharlotteFrancis It looks like you somehow overrode a merge-conflict and commited this file. Can you fix the merge conflict and resend?